sources/engine/Stride.Physics/Shapes/HeightfieldColliderShape.cs

```diff
@@ -14,25 +14,6 @@
 
 namespace Stride.Physics
 {
-    namespace Shapes
-    {
-        [Obsolete("This class will be deprecated. Use 'Stride.Physics.HeightfieldColliderShape'.", false)]
-        public class HeightfieldColliderShape : Stride.Physics.HeightfieldColliderShape
```

Comment:
Did you check if this was used in any samples?

Reply:
@Kryptos-FR I checked this by searching the entire stride folder using CTR +F and found no direct reference for any of the samples.
Just in case, I tested some samples: PhysicsSample,  ThirdPersonShooter, TopDownRPG.
No regression so far 😃